README.md

```diff
@@ -76,102 +48,160 @@ Source: [sajari.css](./sample-styles/sajari.css)
 
 ![Sajari](./sample-styles/sajari.png)
 
-## Configuration
 
-The generated search interfaces are configured using a simple JSON object which contains attributes that control:
+## Integrations
 
-* [Project/Collection](#projectcollection)
-* [Pipeline](#pipeline)
-* [Attaching to the DOM](#attaching-to-the-dom)
-* [Result Config](#result-config)
-* [Search box place holder text](#search-box-placeholder-text)
-* [Algorithm parameters](#algorithm-parameters)
-* [Tab filters](#tab-filters)
+There are 4 types of integration:
 
-You'll find the configuration object in the snippet generated from the [install page](https://www.sajari.com/console/collections/install).
+* [*inline*](#inline) (search box, results).  Interface is embedded directly into a page (or pages) on your website, for instance a dedicated search page with a search box + results.
 
-### Project/Collection
+* [*overlay*](#overlay) (full page overlay, search box, results).  Interface appears as an overlay on top of the current page.  Can be used to search without leaving the page.
 
-The `project` and `collection` attributes set which project/collection combo to query.  These can be found in the Console.
+* [*search box*](#search-box) (search box).  Typical usage includes being embedded into headers and menus.
 
-```javascript
-project: "your-project",
-collection: "your-collection",
-```
+* [*dynamic content*](#dynamic-content) (results). Typically used to show similar or popular content.
 
-### Pipeline
+It's possible to use multiple integrations, for instance: have a *searchbox* in the header of your site, which then redirects to an *inline* search results page when triggered.
```

Comment:
It's possible to use multiple integrations on the same site/page.  For instance:

Reply:
I would put the details in a seperate section Multiple integrations (or similar).